site/en/docs/extensions/mv3/migrating_to_service_workers/index.md

```diff
@@ -169,46 +186,48 @@ chrome.alarms.onAlarm.addListener(() => {
-to access the [DOMParser][9] API or create iframes to parse and traverse documents. Extension
-developers have two ways to work around this limitation: create a new tab or use a library. Which
-you choose will depend on your use case.
+to access the [`DOMParser`][9] API or create an `<iframe>` to parse and traverse documents.
+Extension developers have two ways to work around this limitation: create a new tab or use a
+library. Which you choose will depend on your use case.
 
-Libraries such as [jsdom][10] can be used to emulate a typical browser window environment, complete
-with DOMParser, event propagation, and other capabilities like requestAnimationFrame. Lighter weight
-alternatives like [undom][11] provide just enough DOM to power many frontend frameworks and
-libraries.
+Libraries such as [`jsdom`][10] can be used to emulate a typical browser window environment,
+complete with DOMParser, event propagation, and other capabilities like `requestAnimationFrame`.
+Lighter-weight alternatives like [`undom`][11] provide just enough DOM to power many frontend
+frameworks and libraries.
 
-Extensions that need a full native browser environment can use the [chrome.windows.create()][12] and
-[chrome.tabs.create()][13] APIs from inside a service worker. Additionally, an extension's popup
-still provides a full (temporary) window environment.
+Extensions that need a full native browser environment can use the [`chrome.windows.create()`][12]
+and [`chrome.tabs.create()`][13] APIs from inside a service worker to create a real browser window. 
+Additionally, an extension's popup still provides a full (temporary) window environment.
 
 ### Audio/video playback and capture {: #audio_vidio }
 
 It's not currently possible to play or capture media directly in a service worker. In order for a
 Manifest V3 extension to leverage the web's media playback and capture capabilities, the extension
-will need to create a window environment using either [chrome.windows.create()][12] or
-[chrome.tabs.create()][13]. Once created, the extension can use [message passing][16] to coordinate
-between the playback document and service worker.
+will need to create a window environment using either [`chrome.windows.create()`][12] or
+[`chrome.tabs.create()`][13]. Once created, the extension can use [message passing][16] to
+coordinate between the playback document and service worker.
 
 ### Rendering to a canvas {: #canvas }
 
 In some cases developers use background pages to render content for display in other contexts or to
-create and cache assets. While service workers don't have access to DOM and therefore cannot use
-`<canvas>` elements, service workers do have access to the [OffscreenCanvas API][17].
+create and cache assets. While Service Wrkers don't have access to DOM and therefore cannot use
+`<canvas>` elements, service workers do have access to the [Offscreen Canvas API][17].
```

Comment:
Why OffscreenCanvas --> Offscreen Canvas? Isn't it an API name? And if not, why capitalized?